src/components/Layout/Layout.scss

```diff
@@ -13,11 +13,30 @@ $relative-size: $primary-basis / $secondary-basis;
   align-items: flex-start;
   margin-top: -1 * spacing();
   margin-left: -1 * spacing(loose);
+
+  @media print {
+    padding: 0;
+    margin-top: -1 * spacing();
```

Comment:
Wouldn't this affect every layout component in use? Just confirming this broad change isn't too risky outside the scope we're working towards on our specific work that prompted this change :)

Reply:
Just tested and these are redundant, inherited from the Orders print styles. Actually [a coupe of lines above](https://github.com/Shopify/polaris-react/pull/4142/files#diff-5d39bc98db083e35f0f49470a0e013f5f91b3be3b890d0cde5af37718f85fcefR14) there are the same styles for the `.Layout` component without print styles. Well spotted!